runtime/interop/genesis.go

```diff
@@ -126,14 +126,18 @@ func GethPragueTime(genesisTime uint64, cfg *clparams.BeaconChainConfig) *uint64
 // like in an e2e test. The parameters are minimal but the full value is returned unmarshaled so that it can be
 // customized as desired.
 func GethTestnetGenesis(genesisTime uint64, cfg *clparams.BeaconChainConfig) *core.Genesis {
-	ttd, ok := big.NewInt(0).SetString(clparams.BeaconConfig().TerminalTotalDifficulty, 10)
-	if !ok {
-		panic(fmt.Sprintf("unable to parse TerminalTotalDifficulty as an integer = %s", clparams.BeaconConfig().TerminalTotalDifficulty))
-	}
-
 	shanghaiTime := GethShanghaiTime(genesisTime, cfg)
+	if cfg.CapellaForkEpoch == 0 {
```

Comment:
wondering if this is the best way to handle this instead of updating the function above. it's used elsewhere so perhaps it's not good as it changes the behavior